src/components/Nav/Nav.js

```diff
@@ -1,11 +1,29 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { AiOutlineSearch } from 'react-icons/ai';
 import Dropdown from './Dropdown/Dropdown';
+import Cart from './Cart/Cart';
 import './Nav.scss';
 
 export default function Nav() {
   const [searchActivated, setSearchActivated] = useState(false);
+  const [cartValue, setCartValue] = useState([]);
+  const [classOfCartWrap, setClassOfCartWrap] = useState('');
+  // todo : 장바구니 수량 state로 관리하고, +-버튼 눌렀을 때 setState
+  // todo : 적용버튼 클릭 시 fetch로 변경 요청
+
+  useEffect(() => {
+    fetch(`http://b474-211-106-114-186.ngrok.io/carts/list/2`)
+      .then(res => res.json())
+      .then(res => {
+        setCartValue(res);
+      });
+  }, []);
+
+  const activateCart = () => {
+    if (classOfCartWrap === '') setClassOfCartWrap('activated');
+    if (classOfCartWrap === 'activated') setClassOfCartWrap('');
```

Comment:
이렇게 작성하면 반복되는 구문을 줄일 수 있겠네요!
```suggestion
    setClassOfCartWrap(classOfCartWrap === 'activated' ? '' : 'activated');
```